### PR TITLE
Remove useless TODO comment

### PR DESCRIPTION
**Pull request by bronzdoc**
# Description:
Without context this comment is useless, so just removing it...

______________

I will abide by the [code of conduct](https://github.com/rubygems/rubygems/blob/master/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: lib/rubygems/specification.rb [hunk 1]
```diff
@@ -1530,7 +1530,7 @@ def authors
   # a full path.
 
   def bin_dir
-    @bin_dir ||= File.join gem_dir, bindir # TODO: this is unfortunate
```
Comment (simi) [inline lib/rubygems/specification.rb]: Do you think that TODO it related to the method NOTE and possible confusion?

Comment (bronzdoc) [reply lib/rubygems/specification.rb]: I thik it is!, thanks for pointing that out @simi 

Maybe rename `bin_dir` to `bin_dir_path` ?

Comment (simi) [reply lib/rubygems/specification.rb]: Uhh, I think changing public specification method signature is not good idea. Anyway `NOTE` is probably enough in here. There's no need for this TODO.

**Comment by bronzdoc** [comment]
@bundlerbot r+

File: lib/rubygems/specification.rb (hunk 1 above)
Comment (bronzdoc) [reply lib/rubygems/specification.rb]: I was not clear... Yeah, I meant as suggestion for the future

**Pull request merged**